src/imageio/format/jpeg.c

```diff
@@ -396,6 +394,10 @@ int write_image(dt_imageio_module_data_t *jpg_tmp, const char *filename, const v
   jpeg_finish_compress(&(jpg->cinfo));
   jpeg_destroy_compress(&(jpg->cinfo));
   fclose(f);
+
+  if(exif && exif_len > 0 && exif_len < 65534)
```

Comment:
It was done like that before. Initialy I thought it would make sense to keep the test, but then, I found some case (working on jpegs and exporting them) where darktable lenght test would be OK when exiv2 internal one would not, so this is indeed superfluous.

Do you want me to make a pull request to track the issue?